ci-operator/config/openshift/lightspeed-console/openshift-lightspeed-console-main.yaml

```diff
@@ -51,6 +51,13 @@ tests:
   optional: true
   steps:
     workflow: openshift-ci-security
+- as: fips-image-scan-service
+  skip_if_only_changed: ^docs/|\.md$|lightspeed-catalog.*|^\.tekton/|^\.github/|^hack/|^(?:.*/)?(?:\.gitignore|\.snyk|\.dockerignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
console does not have this `lightspeed-catalog.*` but it does not harm

Reply:
updated